features/fixtures/EDM_Fixture/Assets/ExternalDependencyManager/Editor/LICENSE

```diff
@@ -0,0 +1,245 @@
+Copyright (C) 2014 Google Inc.
```

Comment:
Is it feasible to curl this dependency when we run the test rather than check it in? This makes the version we're testing against a little more explicit - potentially always testing against the [latest](https://github.com/googlesamples/unity-jar-resolver/blob/master/external-dependency-manager-latest.unitypackage)?

Reply:
yes i think that's a good idea, i'll make a note in the CI task for when we get it all setup.